packages/core/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "CSS for Components",
   "main": "dist/index.js",
   "scripts": {
-    "test": "mocha \"./dist/test/**/*.spec.js\"",
-    "start": "webpack serve"
+    "test": "mocha ./dist/test/**/*.spec.js",
+    "start": "mocha-play ./dist/test/**/*.spec.js -c ./webpack.config.js -w"
```

Comment:
Call it test:browser and run it in test script, also running test:node

Reply:
this will be done in a separate PR